wagtail_streamfield_migration_toolkit/autodetect/comparers.py

```diff
@@ -30,22 +30,20 @@ def compare(cls, old_def, old_name, new_def, new_name):
 
         # TODO it might be best to add some separate tests for the hashable_deep_deconstruct method
         # itself
-        old_path, _, old_kwargs = cls.hashable_deep_deconstruct(old_def)
-        new_path, _, new_kwargs = cls.hashable_deep_deconstruct(new_def)
+        old_path, _, old_options = cls.hashable_deep_deconstruct(old_def)
+        new_path, _, new_options = cls.hashable_deep_deconstruct(new_def)
```

Comment:
I would stick with `kwargs` rather than `options` here - at this stage it really is the "parameters that were passed as keyword arguments"